spec/controllers/projects_controller_spec.rb

```diff
@@ -158,6 +158,20 @@
     end
   end
 
+  describe "#toggle_locked" do
+    context "when locking a project" do
+      it "returns a datetime" do
+        patch :toggle_locked, params: {id: project.id}, xhr: true
+        expect(assigns[:project]).to be_locked
+      end
+
+      it "returns a js response" do
```

Comment:
Why is it important that we test the response type?

Reply:
if someone wants to hit this endpoint with an html or json request, they will notice that we are only responding within `js` format for now. what do you think?

Reply:
Good point 👍